index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>MrBlue</title>
+	<link rel="stylesheet" href="static/style.css">
+	<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+</head>
+<body>
+	<header class="header">
+		<img src="static/logo.png" alt="" srcset="">
+     
+      <nav class="menu" id="menu">    
+        <div class="menu-close">
+            <a href="#" class="menu-close-icon"><img src="static/ic_menu.svg" alt="" srcset=""></a>
+        </div>
+        </nav>
+
+    </header>
+
+	<main>
+        <section class="bio">
+            <p class="bio-header">
+                Bleu Yves <br> Sopoude<br>
+                
+	
+            </p>
```

Comment:
Because this stands for the main title of the website, kindly use a h1 tag instead of a p tag.

If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.